recipes-omnect/omnect-health-check/omnect-health-check/omnect_service_log.tmpfilesd

```diff
@@ -0,0 +1,7 @@
+# Note:
+#   the log directory below can only be written by privileged prozesses, so
+#   any scripts wanting to do so need suitable permissions.
+#   as currently only omnect_service_log.sh is writing to this directory and
+#   only in the context of systemd hooks (PreStart/StopPost) it is sufficient
+#   to instruct systemd to let it run privileged.
+d /run/omnect_health_log 0775 root root -
```

Comment:
imho could also be part of /run/omnect-device-service or be renamed to omnect-health-check

Reply:
done